app/jobs/v3/update_service_instance_job.rb

```diff
@@ -62,6 +52,22 @@ def perform
           })
           raise err
         end
+
+        if service_instance.last_operation.state == 'failed'
+          logger.info("Service instance update failed: #{service_instance.last_operation.description}")
+          operation_failed!(service_instance.last_operation.description)
+        end
+
+        logger.info("Service instance update complete #{service_instance_guid}")
```

Comment:
Is the update really complete, or is it just this job?

Reply:
thanks for spotting this.